doc/api-extensions.md

```diff
@@ -2599,3 +2599,9 @@ The following pool level configuration keys have been added:
 1. {config:option}`storage-pure-pool-conf:pure.api.token`
 1. {config:option}`storage-pure-pool-conf:pure.mode`
 1. {config:option}`storage-pure-pool-conf:pure.target`
+
+## `cloud_init_ssh_keys`
+
+Adds support for injecting additional SSH public keys on instances through `cloud-init` without conflicting with any configuration present on `cloud-init.vendor-data` or `cloud-init.user-data`.
+
+To achieve this, the `cloud-init.ssh-keys.KEYNAME` configuration key is added for both instances and profiles, that can be used for specifying one desired additional public key to be injected. Where `KEYNAME` can be any arbitrary name for the injected key.
```

Comment:
Please can you document here what the value of this key should be too (as well as in the docs).